config/mv_checks.xml

```diff
@@ -192,7 +192,6 @@
         <module name="PackageDeclaration"/>
         <module name="ParameterAssignment"/>
         <module name="RequireThis"/>
-        <module name="ReturnCount"/>
```

Comment:
Can we just increase the number of permitted returns? Having lots of returns isn't great...